src/test/java/com/faforever/client/game/GameRangePredicateTest.java

```diff
@@ -0,0 +1,130 @@
+package com.faforever.client.game;
+
+import com.faforever.client.player.Player;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GameRangePredicateTest {
+
+  private Player validPlayer;
+  private GameRangePredicate predicate;
+  private Game game;
+
+  @Before
+  public void setUp() {
+    validPlayer = new Player("");
```

Comment:
I think there is a PlayerBuilder

Reply:
For these tests only the rating is relevant so don't think using the player builder gives any value. Do you disagree?